README.md

```diff
@@ -0,0 +1,22 @@
+# QSL Buildscript Prototype
```

Comment:
if you're going to start implementing QSL proper this is no longer a buildscript prototype